src/bot/cogs/image_commands.py

```diff
@@ -217,6 +217,26 @@ async def pics(self, ctx: commands.Context):
 
         ctx.reply(embed=embed)
 
+    def update_channel_enabled(self, ctx: commands.Context, *, enabled: bool):
+        db_channel = self.db.get_channel_by_discord_id(ctx.channel.id)
+
+        self.db.update_channel(
+            id=db_channel.id,
+            discord_id=db_channel.discord_id,
+            enabled=enabled,
+            server_id=db_channel.server_id,
+        )
+
+    @app_commands.command(name="enable", description="Enable CordPics in this channel")
+    async def enable(self, ctx: commands.Context):
```

Comment:
```suggestion
    async def enable(self, interaction: discord.Interaction):
```
This wouldn't be `ctx`

Reply:
Thanks for pointing that out, I probably wouldn't have noticed myself 😅
Should be fixed now